learn/basics/start-with-acceptance.md

```diff
@@ -138,3 +138,26 @@ In the simplest case, your "Arrange" step is merely a case of calling one or mor
 That said, while this is the ideal it may not be practical or possible.
 
 Aim to have your test setup code mimic how you'd expect your application to be used by it's delivery mechanism.
+
+## Should we use Code or a Domain Specific Language?
+
+Gherkin (Cucumber/SpecFlow) and Fitnesse are common DSL choices for writing executable acceptance tests.
+
+If you are involving your (non-programmer) stakeholders in creation and verification of acceptance tests, you should probably use a DSL. If you are not doing this, use code. 
```

Comment:
```suggestion
If you are involving your (non-programmer) stakeholders in creation and verification of acceptance tests, you should probably use a DSL. If you are not doing this, use code, but try to still use human-readable language. 
```

Not sure if "human-readable" is the right way to express this, but something like this might be good?

Reply:
Yea I think "language that does not talk about delivery mechanism details " + "use the language you would use in discussions with non-technical stakeholders"